contrib/loaddata.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cat takwimu/sql/*.sql | psql            # Load tables / data into DB (TBD)
+python manage.py loaddata takwimu/fixtures/takwimu_wazimap_fixtures.json # Load fixtures
```

Comment:
Just a question, in all HURUmap-apps, we just call this file the name of the app e.g. in HURUmap/UG, this is called `hurumap_ug.json`. Why aren't we calling this one just `takwimu.json` or if we want to stress that they're for wazimap, why don't we just call them that, i.e. `wazimap.json`?